gvr-accessibility/app/src/main/assets/gvr.xml

```diff
@@ -28,9 +28,10 @@
         framebufferPixelsHigh="DEFAULT"
         framebufferPixelsWide="DEFAULT"
         showLoadingIcon="true"
-        useMultiview="true"
+        useMultiview="false"
```

Comment:
Why multiview needs to be turned off?

Reply:
It doesn't. I will turn it back on. I was testing both cases.